### DISCUSSION TITLE
Dual GPU support working?

### DISCUSSION THREAD

**Discussion by fliberdygibits** [AbdBarho/stable-diffusion-webui-docker#263] [Q&A]
I've got a tesla K80 passed to docker.  Both GPUs are visible inside the container but SD only sees on in the settings.  Is there something extra I need to do or is it only single GPU currently?

**Comment by AbdBarho** [comment]
Which UI are you using? you may have to specify additional cli args to tell the ui which gpu to use.

As far as I know, you can't use both gpus simultaneously to generate images, but maybe I am mistaken.

**Comment by fliberdygibits** [comment]
Sygil-sl I think.  Is it just not capable of using 2 GPUs even if it sees them?  Also running in docker, what cli args are we talking about?  the docker run command used to spin it up?

**Reply by AbdBarho** [reply]
See if any of the configs here match your needs:

https://github.com/Sygil-Dev/sygil-webui/blob/master/configs/webui/webui_streamlit.yaml

you can then overwrite this file with what you want:
https://github.com/AbdBarho/stable-diffusion-webui-docker/blob/master/services/sygil/userconfig_streamlit.yaml

We should make configuration much more flexible than this, but there has been no demand for it.
